tests/personal_background/test_database_model.py

```diff
@@ -0,0 +1,139 @@
+import unittest
+from tests.base_test_case import BaseTestCase
+from app.utils.bitschema_utils import (
+    Gender,
+    Age,
+    Ethnicity,
+    SexualOrientation,
+    Religion,
+    PhysicalAbility,
+    MentalAbility,
+    SocioEconomic,
+    HighestEducation,
+    YearsOfExperience,
+)
+from app.database.models.user import UserModel
+from app.database.models.personal_background import PersonalBackgroundModel
+from app.database.sqlalchemy_extension import db
+from app.database.db_types.JsonCustomType import JsonCustomType
+from tests.test_data import user1, user2
+
+# Testing User's Personal background database model
+class TestPersonalBackgroundModel(BaseTestCase):
+    def setUp(self):
+        super().setUp()
+
+        # Add a user1 into the database
+        user_1 = UserModel(
+            name=user1["name"],
+            email=user1["email"],
+            username=user1["username"],
+            password=user1["password"],
+            terms_and_conditions_checked=user1["terms_and_conditions_checked"],
+        )
+
+        # Add a user2 into the database
+        user_2 = UserModel(
+            name=user2["name"],
+            email=user2["email"],
+            username=user2["username"],
+            password=user2["password"],
+            terms_and_conditions_checked=user2["terms_and_conditions_checked"],
+        )
+
+        db.session.add(user_1)
+        db.session.add(user_2)
+        db.session.commit()
+
+        self.user1 = UserModel.query.filter_by(email=user_1.email).first()
+        self.user2 = UserModel.query.filter_by(email=user_2.email).first()
+
+        # add user2 personal_background
+        self.user2_background = PersonalBackgroundModel(
+            user_id=self.user2.id,
+            gender=Gender.FEMALE,
+            age=Age.AGE_25_TO_34,
+            ethnicity=Ethnicity.CAUCASIAN,
+            sexual_orientation=SexualOrientation.LGBTQIA,
+            religion=Religion.OTHER,
+            physical_ability=PhysicalAbility.WITHOUT_DISABILITY,
+            mental_ability=MentalAbility.WITHOUT_DISORDER,
+            socio_economic=SocioEconomic.LOWER_MIDDLE,
+            highest_education=HighestEducation.BACHELOR,
+            years_of_experience=YearsOfExperience.UP_TO_10,
+        )
+        self.user2_background.others = {"religion": "Daoism"}
+        self.user2_background.is_public = True
+
+        db.session.add(self.user2_background)
+        db.session.commit()
+
+    def test_user1_background_not_exist(self):
+
+        user1_background = PersonalBackgroundModel.query.filter_by(
+            user_id=self.user1.id
+        ).first()
+        self.assertTrue(user1_background is None)
+
+    def test_user2_background_creation(self):
+        background = PersonalBackgroundModel.query.filter_by(
+            user_id=self.user2.id
+        ).first()
+        self.assertTrue(background is not None)
```

Comment:
just as a fyi, you can do also `assertIsNotNone()` I think it's like this 🤔 
not that you need to change, just wanted to let you know ;)

Reply:
I forgot about that 🤣 . Thanks @isabelcosta for the reminder. I've fixed it now 😉 .